node/tests/hubController.js

```diff
@@ -0,0 +1,45 @@
+const sleep = require('es6-sleep').promise;
+var test = require('ava');
+var config = require('./hubController-testConfig');
+
+var hubControllerPath = require('path').join(__dirname, '..');
+var HubController = require("../hubController");
```

Comment:
>HubController [](start = 4, length = 13)

nit: naming - is there a way to distinguish between the module and the object of type hubController?